pytorch_lightning/lite/lite.py

```diff
@@ -410,7 +410,13 @@ def _run_impl(self, run_method: Callable, *args: Any, **kwargs: Any) -> Any:
     def _run_with_strategy_setup(self, run_method: Callable, *args: Any, **kwargs: Any) -> Any:
         self._strategy.setup_environment()
         with self._strategy.model_sharded_context(), _replace_dataloader_init_method():
-            return run_method(*args, **kwargs)
+            result = run_method(*args, **kwargs)
+
+        # TODO: Capture this inside a teardown
+        if isinstance(self._strategy, BaguaStrategy) and self._strategy._bagua_algorithm == "async":
+            self._strategy.model.module.bagua_algorithm.abort(self._strategy.model.module)
```

Comment:
This is required for the async method. Does that mean we would need a self.teardown() method that the user calls inside Lite? Or do we need to drop support for async in Lite?